misc/config_tools/xforms/lib.xsl

```diff
@@ -0,0 +1,495 @@
+<?xml version='1.0' encoding='utf-8'?>
+<xsl:stylesheet
+    version="1.0"
+    xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+    xmlns:dyn="http://exslt.org/dynamic"
+    xmlns:math="http://exslt.org/math"
+    xmlns:func="http://exslt.org/functions"
+    xmlns:acrn="http://projectacrn.org"
+    extension-element-prefixes="func">
+
+  <xsl:variable name="digits" select="'0123456789abcdefghijklmnopqrstuvwxyz'" />
```

Comment:
It can be shorten, I am fine with that.

Regarding to the error, I think that should be handle in schema instead here. xform is to translate the valid data to desired output format.

Reply:
removed ghijklmnopqrstuvwxyz.